src/core/gui/menus/menubar/RecentDocumentsSubmenu.cpp

```diff
@@ -39,7 +39,7 @@ auto createRecentMenuItem(const GtkRecentInfo* info, size_t i) {
 
     // escape underscore
     StringUtils::replaceAllChars(display_name, {replace_pair('_', "__")});
-    std::string label = FS(FORMAT_STR("{1}. {2}") % (i + 1) % display_name);
+    std::string label = FS(FORMAT_STR("{1}. {2}") % (as_signed(i) + 1) % display_name);
```

Comment:
I tried adding 
```c++ 
PlaceholderString& operator%(uint64_t value);
```
But this confused the compiler, since it didn't know what to do with an `uint32_t` then. But I agree, there should be a solution to that.